Src/Base/AMReX_Random.cpp

```diff
@@ -98,10 +98,10 @@ InitRandom (ULong cpu_seed, int nprocs, ULong gpu_seed)
 #endif
 
 #ifdef AMREX_USE_OMP
-#pragma omp parallel
+#pragma omp parallel for
 #endif
+    for (int tid = 0; tid < nthreads; tid++) {
```

Comment:
```suggestion
    for (int tid = 0; tid < nthreads; tid++)
```
With the for loop, the parallel region isn't even required anymore.

Reply:
Yes, I was thinking the same. Not sure it matters...